src/cli/bootstrap/bootstrap.ts

```diff
@@ -46,25 +45,36 @@ function bytesToTokens(bytes: number): string {
     return ((bytes * (1 + fee)) / ramPrice).toFixed(4) + ` ${CURRENCY_SYMBOL}`;
 }
 
-export default async function bootstrap(args: string[]) {
-    if (!args[0]) throw new Error('Missing public key argument');
+const signer = getSigner();
 
+export default async function bootstrap() {
     try {
-        const newPrivateKey = PrivateKey.from(args[0]);
+        if (!process.env.TONOMY_OPS_PRIVATE_KEY) throw new Error('Missing TONOMY_OPS_PRIVATE_KEY');
```

Comment:
Here we need to add env and added keys there which created during easy cleos script?

Reply:
easycleos will be used for the board keys, and only for the Pangea mainnet. The Ops key will be managed by us developers